<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterpalette</class>
 <widget class="QDialog" name="ajouterpalette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="IdPalette">
     <property name="text">
      <string>IdPalette :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="IdPalette_2"/>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="checkBox_IdAutomatique">
     <property name="text">
      <string>Automatique</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Destination">
     <property name="text">
      <string>Destination :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="Destination_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="DateEnvoiPrevue">
     <property name="text">
      <string>DateEnvoiPrevue :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="DateEnvoiPrevue_2"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="CapaciteMax">
     <property name="text">
      <string>CapaciteMax :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="CapaciteMax_2"/>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Produits</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="TypeProduits">
        <property name="text">
         <string>Type Produit :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_TypeProduits"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableProduits"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="Creer">
     <property name="text">
      <string>Cr√©er</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="Annuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
